README.md

```diff
@@ -50,6 +50,26 @@ Divvy allows users to enter expenses as they are incurred, and calculates the be
 
 ### Installing Dependencies (tbd)
 
+### Setting Up passport-facebook
+
+1. Sign up to be a [facebook developer](https://developers.facebook.com/).
+2. Click the "Add a New App" on your [apps page](https://developers.facebook.com/apps/).
+3. Choose a "Website" app and give it a name.
+4. Add a contact email and a category when prompted.
+5. Complete the captcha and wait for the app to be created.
+6. Under the "Tell us about your website" portion of the next page, enter:  
```

Comment:
You need to scroll down to the "Tell us about your website" portion, while the "Skip Quick Start" button is visible in the upper right hand corner of prominent dark grey "Quick Start for Website" box.